main.go

```diff
@@ -190,6 +194,13 @@ func main() {
 		server.PrintAndDie(fmt.Sprintf("%s: %s", exe, err))
 	}
 
+	if errCreateUsers != nil {
```

Comment:
Add a comment that explain why this logs are here and not below the function creating the users